docs/integration-suse-ai.adoc

```diff
@@ -0,0 +1,221 @@
+
+Refer to the official https://documentation.suse.com/suse-ai/1.0/[SUSE AI documentation] for detailed information. In this guide, we will focus on deploying the `Trento MCP Server` using SUSE AI, specifically with Ollama and Open Web UI. Always refer to the latest instructions in the https://documentation.suse.com/suse-ai/1.0/html/AI-deployment-intro/index.html[SUSE AI deployment guide] for the most accurate and up-to-date information.
+
+=== Prerequisites
+
+This guide assumes:
+
+* you have a Kubernetes cluster set up and running (with an ingress controller and cert-manager installed)
+* you have access to the SUSE Application Collection.
+* if you want to run Ollama models, you have either:
+** a Kubernetes cluster with sufficient resources.
+** a cloud provider and enough permissions to deploy Ollama models remotely.
+
+==== Limitations
+
+Deploying the entire SUSE AI stack requires plenty of resources, especially for running Ollama models. Alternatively, you can deploy only the Open Web UI and connect it to an Ollama instance running elsewhere. This is the approach we will take here: on-premises Open Web UI with Ollama running on a remote server (e.g., Google Cloud).
+
+==== Getting the artifacts from SUSE Application Collection
+
+You need access to SUSE Application Collection and the proper entitlements to download the required artifacts. Always refer to the https://docs.apps.rancher.io/get-started/authentication/[SUSE Application Collection documentation] for the latest instructions on how to authenticate and access the collection. This guide assumes you have the necessary credentials and access to the collection.
+
+____
+NOTE: If you want to run the entire stack, you can also use the https://github.com/SUSE/suse-ai-deployer[SUSE AI Deployer Helm Chart].
+____
+
+[arabic]
+. Login to the SUSE Application Collection:
+
+[source,console]
+----
+# Replace REPLACE_WITH_YOUR_USERNAME and REPLACE_WITH_YOUR_PASSWORD with your actual credentials
+helm registry login dp.apps.rancher.io/charts -u <REPLACE_WITH_YOUR_USERNAME@apps.rancher.io> -p REPLACE_WITH_YOUR_PASSWORD
+----
+
+[arabic]
+. Create a namespace for SUSE AI
+
+[source,console]
+----
+kubectl create namespace suse-ai
+----
+
+[arabic]
+. Create a Kubernetes Pull Secret (`application-collection`) for the SUSE Application Collection:
+
+[source,console]
+----
+# Replace REPLACE_WITH_YOUR_USERNAME and REPLACE_WITH_YOUR_PASSWORD with your actual credentials
+kubectl create -n suse-ai secret docker-registry application-collection --docker-server=dp.apps.rancher.io --docker-username=<REPLACE_WITH_YOUR_USERNAME@apps.rancher.io> --docker-password=REPLACE_WITH_YOUR_PASSWORD
+----
+
+==== Install Open Web UI
+
+This section describes how to install the Open Web UI, which provides a user-friendly interface for interacting with AI models.
+
+[arabic]
+. Create a link:./examples/values.openwebui.yaml[values.openwebui.yaml] file with the values for Open Web UI:
```

Comment:
Having the example and the link is a bit much maintenance for the future, could you consider to stick to one ?

Reply:
ditto, good point, thx!